doc/source/management/innodb_expanded_fast_index_creation.rst

```diff
@@ -4,16 +4,10 @@
 Expanded Fast Index Creation
 ============================
 
-Percona has implemented several changes related to |MySQL|'s fast index creation feature. This feature expands the ``ALTER TABLE`` command by adding a new clause that provides online index renaming capability, that is renaming indexes without rebuilding the whole table.
-
-Enabling Expanded Fast Index Creation
-=====================================
-
-Fast index creation was implemented in |MySQL| as a way to speed up the process of adding or dropping indexes on tables with many rows. However, cases have been found in which fast index creation creates an inconsistency between |MySQL| and |InnoDB| data dictionaries.
+Percona has implemented several changes related to |MySQL|'s fast index creation feature. Fast index creation was implemented in |MySQL| as a way to speed up the process of adding or dropping indexes on tables with many rows. However, cases have been found in which fast index creation creates an inconsistency between |MySQL| and |InnoDB| data dictionaries.
```

Comment:
Hmm what cases? If nobody knows, maybe we can remove this clause and wait for bug reports?